packages/components/src/components/single-select/shadow.tsx

```diff
@@ -0,0 +1,664 @@
+									required={this.state._required}
+									spellcheck="false"
+									{...this.controller.onFacade}
+									onInput={this.onInput.bind(this)}
+									onChange={this.onChange.bind(this)}
+									placeholder={this.state._placeholder}
+									onClick={this.toggleListbox.bind(this)}
+									onBlur={() => {
+										this.handleInputValue();
+									}}
+								/>
+								{this._inputValue && (
+									<KolIconTag
+										_icons="codicon codicon-close"
+										_label={translate('kol-delete-selection')}
+										onClick={() => {
+											this.clearSelection();
+											this.refInput?.focus();
+										}}
+										class="single-select__delete"
+									/>
+								)}
+
+								<KolButtonWcTag
+									class="single-select__button"
+									_label={translate('kol-dropdown')}
+									_variant="ghost"
+									_disabled={this.state._disabled}
+									_on={{
+										onClick: (event: Event): void => {
+											this.toggleListbox(event);
+										},
+									}}
+									_hideLabel
+									_icons="codicon codicon-triangle-down"
+								></KolButtonWcTag>
+							</div>
+							{this._isOpen && !(this.state._disabled === true) && (
+								<ul role="listbox" class="single-select__listbox" onKeyDown={this.handleKeyDownDropdown.bind(this)}>
+									{Array.isArray(this._filteredOptions) && this._filteredOptions.length > 0 ? (
+										this._filteredOptions.map((option, index) => (
+											<li
+												id={`option-${index}`}
+												key={`-${index}`}
+												ref={(el) => {
+													if (el) this.refOptions[index] = el;
+												}}
+												tabIndex={0}
+												role="option"
+												aria-selected={this.state._value === (option as Option<string>).value}
+												onClick={(event: Event) => {
+													this.selectOption(event, option as Option<string>);
+													this.refInput?.focus();
+													this.toggleListbox(event);
+												}}
+												onMouseOver={() => {
+													this._focusedOptionIndex = index;
+													this.focusOption(index);
+												}}
+												onFocus={() => {
+													this._focusedOptionIndex = index;
+													this.focusOption(index);
+												}}
+												class="single-select__item"
+												onKeyDown={(e) => {
+													if (e.key === 'Enter' || e.key === 'NumpadEnter') {
+														this.selectOption(e, option as Option<string>);
+														this.refInput?.focus();
+														e.preventDefault();
+													}
+												}}
+											>
+												<input
+													class="visually-hidden"
+													type="radio"
+													name="options"
+													id={`option-radio-${index}`}
+													value={(option as Option<string>).value}
+													checked={this.state._value === (option as Option<string>).value || index === this._focusedOptionIndex}
+												/>
+
+												<label htmlfor={`option-radio-${index}`} class="radio-label">
```

Comment:
```suggestion
												<label for={`option-radio-${index}`} class="radio-label">
```
(`htmlfor` gibt es in Stencil nicht)